src/config/config.ts

```diff
@@ -0,0 +1,44 @@
+// import Web3 from "web3";
+// import MemoryStore from "../storage/memoryStorage";
```

Comment:
maybe we can use node-convict for configuration?

Reply:
sure, I think it's worth a research spike. would you mind adding a chore to the backlog?

Reply:
I took a quick look and I liked what I saw. Added it to the backlog to discuss https://www.pivotaltracker.com/story/show/177682091

Reply:
Same here, looks good at first glance